sdk/eventhub/Microsoft.Azure.WebJobs.Extensions.EventHubs/src/EventHubTriggerAttribute.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.Azure.WebJobs.Description;
+
+namespace Microsoft.Azure.WebJobs
+{
+    /// <summary>
+    /// Setup an 'trigger' on a parameter to listen on events from an event hub.
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Parameter)]
+    [Binding]
+    public sealed class EventHubTriggerAttribute : Attribute
+    {
+        /// <summary>
+        /// Create an instance of this attribute.
+        /// </summary>
+        /// <param name="eventHubName">Event hub to listen on for messages. </param>
+        public EventHubTriggerAttribute(string eventHubName)
+        {
+            EventHubName = eventHubName;
+        }
+
+        /// <summary>
+        /// Name of the event hub.
+        /// </summary>
+        public string EventHubName { get; private set; }
+
+        /// <summary>
+        /// Optional Name of the consumer group. If missing, then use the default name, "$Default"
+        /// </summary>
+        public string ConsumerGroup { get; set; }
+
+        /// <summary>
+        /// Gets or sets the optional app setting name that contains the Event Hub connection string. If missing, tries to use a registered event hub receiver.
+        /// </summary>
+        public string Connection { get; set; }
```

Comment:
Not sure what level of back-compat we're aiming for, but we may want to consider `ConnectionString` instead to avoid confusion with the `EventHubConnection` type.

Reply:
@jsquire Changing property names like this would create a significant amount of downstream work. For example, this property gets set in function.json by the UX in the azure portal when you create an eventhub triggered function in the integrate tab. Same thing when working locally with VS code. So it would only make sense if there was a really significant source of confusion and we were ready to do all the other required work.

Reply:
Fair enough.  Please forgive me ignorance with respect to how these connect to the larger context;  definitely not worth the overhead that the change would cause, in this case.